.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
```

Comment:
maybe we should add some checks on compile?
https://github.com/ec4j/editorconfig-maven-plugin
something like this, by default idea doesn't look on this file, i should install plugin

Reply:
it has only 5 starts - not enterprise levev

Reply:
it's just example
there are no any checks, i can write code in any way
we should add checks or remove this file

Reply:
editorconfig used for development, to have common config for our IDE (it's called EDITORconfig).
It's not a linter.

Reply:
by default ide doesn't use it, isn't it?
i should install a plugin (and i can not install)

Reply:
Yep u should install plugin, it's just for convenience

Reply:
it's not a hard rule and not a linter